clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -325,7 +326,9 @@
     UNSTABLE_OFFSET_COMMIT(88, "There are unstable offsets that need to be cleared.", UnstableOffsetCommitException::new),
     THROTTLING_QUOTA_EXCEEDED(89, "The throttling quota has been exceeded.", ThrottlingQuotaExceededException::new),
     PRODUCER_FENCED(90, "There is a newer producer with the same transactionalId " +
-            "which fences the current one.", ProducerFencedException::new);
+            "which fences the current one.", ProducerFencedException::new),
+    BROKER_AUTHORIZATION_FAILURE(91, "Authorization failed for the request during forwarding. " +
```

Comment:
How about: "A broker failed to authorize itself to another component of the system.  This indicates an internal error on the broker cluster security setup".

This isn't specific to forwarding... there might be other reasons why a broker would need to authorize itself and fail